tests/sse/benchmark_sse1.cpp

```diff
@@ -0,0 +1,154 @@
+// Copyright 2014 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
+// University of Illinois/NCSA Open Source License.  Both these licenses can be
+// found in the LICENSE file.
+
+#include <xmmintrin.h>
+#include "benchmark_sse1.h"
+
+int dst_int = 0;
+
+int main()
+{
+	printf ("{ \"workload\": %u, \"results\": [\n", N);
+	assert(N%4 == 0); // Don't care about the tail for now.
+	float *src = alloc_float_buffer();//(float*)aligned_alloc(16, N*sizeof(float));
```

Comment:
Is the commented-out code here for a reason?

Reply:
Oh, not any more. Nuked that.